File: pset6/mario.py
from cs50 import get_int
l=0

while (l>8 or l<1):
    l=get_int("Height: ")


for i in range(l):
    for j in range(l,i+1,-1):
        print(" ",end="")
    for p in range(0,i+1,1):
        print("#",end="")
    print("  ",end="")
    for q in range(0,i+1,1):
        print("#",end="")
    print()

File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    do{
        height=get_int("Height");
    } while (height <= 0 || height >8);



    for(int i=1; i<=height; i++)
    {
        for(int a=height; a>i; a--)
        {
            printf(" ");
        }

        for(int b=1; b<=i; b++)
        {
            printf("#");
        }

        printf("  ");

        for(int x=1 ; x<=i ; x++)
            {
                 printf("#");

            }
        printf("\n");
    }
}